common/src/main/java/io/pravega/common/util/SimpleCache.java

```diff
@@ -418,5 +418,14 @@ public String toString() {
         }
     }
 
+    public String printCache() {
```

Comment:
Can you find a way to use `ToStringUtils.mapToString` or an overload of that so this is consistent with other code doing this.